linux_sql/sql/ddl.sql

```diff
@@ -1,8 +1,11 @@
+CREATE DATABASE host_agent;
+\c host_agent;
 CREATE TABLE IF NOT EXISTS PUBLIC.host_info (
```

Comment:
There is no need to have PUBLIC.host_info. Only host_info name is enough.

Reply:
Thank you, confused if I needed it or not so I kept it just incase. But will fix it.